index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>iChala Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="header">
+            <h2 class="welcome">Welcome</h2>
+            <img class="tool-button" src="images/Enabled.png" alt="toolimage">
+    </section>
+    <section class="headline">
+  <h1 class="primary-text">Hey there. I’m Ali</h1>
+    <h1 class="green-primary-text">I’m a software developer</h1>
+  <p class="supporting-text">
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+  </p>
+<div class="social-media">
+<a href="https://twitter.com/_ichala" target="_blank" rel="noopener"><img class="social-icon" src="images/social-media-icons/twitter.svg" alt="twitter-icon"></a>
+<a href="https://github.com/ichala/" target="_blank" rel="noopener"><img class="social-icon" src="images/social-media-icons/github.svg" alt="github-icon"></a>
+<a href="https://www.linkedin.com/in/alijendoubi/" target="_blank" rel="noopener"><img class="social-icon" src="images/social-media-icons/Linkedin.svg" alt="Linkedin-icon"></a>
+</div>
```

Comment:
- [ ] Instead of `div` tag please use `ul` tag and `a` tag should be inside of `li` tag. :writing_hand: